distribution/osbuild-composer.socket

```diff
@@ -1,9 +1,10 @@
 [Unit]
-Description=OSBuild Composer API sockets
+Description=OSBuild Composer socket for the weldr API
 
 [Socket]
 ListenStream=/run/weldr/api.socket
-ListenStream=/run/osbuild-composer/job.socket
+SocketGroup=weldr
+SocketMode=660
```

Comment:
Colud we just add these two directives without splitting the socket into two? That would make the rest of the patch redundant (I think). We should still add the user to sysusers though. I think making sure the job socket is not world readable is also the right thing to do. The worker anyway runs as root.

Reply:
Convinced. This makes the patch much simpler. I was concerned about exposing the job socket with the `weldr` group as well, but it doesn't really matter.